controller_manager:
  ros__parameters:
    update_rate: 100

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster

    scara_left_arm_controller:
      type: position_controllers/JointGroupPositionController

    scara_left_gripper_controller:
      type: position_controllers/JointGroupPositionController

    scara_right_arm_controller:
      type: position_controllers/JointGroupPositionController

    scara_right_gripper_controller:
      type: position_controllers/JointGroupPositionController

scara_left_arm_controller:
  ros__parameters:
    joints:
      - scara_left_joint_1
      - scara_left_joint_2

scara_left_gripper_controller:
  ros__parameters:
    joints:
      - scara_left_gripper_joint

scara_right_arm_controller:
  ros__parameters:
    joints:
      - scara_right_joint_1
      - scara_right_joint_2

scara_right_gripper_controller:
  ros__parameters:
    joints:
      - scara_right_gripper_joint
